cmd/lncli/cmd_invoice.go

```diff
@@ -18,7 +18,7 @@ var addInvoiceCommand = cli.Command{
 
 	Invoices without an amount can be created by not supplying any
 	parameters or providing an amount of 0. These invoices allow the payee
-	to specify the amount of satoshis they wish to send.`,
+	to specify the amount of satoshis they wish to receive.`,
```

Comment:
I think this is the wrong way around. If the payee wants to specify the number of satoshis to receive, they use the `--amt` field. If they want to give the option to the other side (the **payer**), then they leave the amount as zero. So this should say "allows the payer ... they with to send".